Description: Skip test that depends on unpackaged optional library
Author: Antonio Terceiro <terceiro@debian.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2015-11-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/blocks_test.rb
+++ b/test/blocks_test.rb
@@ -1278,6 +1278,7 @@ sqrt(3x-1)+(1+x)^2 < y
     end
 
     test 'should render asciimath block in textobject of equation in DocBook backend' do
+      skip 'depends on unpackaged dependency'
       input = <<-'EOS'
 [asciimath]
 ++++
--- a/test/substitutions_test.rb
+++ b/test/substitutions_test.rb
@@ -1374,6 +1374,7 @@ EOS
       #end
 
       test 'should convert asciimath macro content to MathML when asciimath gem is available' do
+        skip 'depends on unpackaged dependency'
         input = 'asciimath:[a < b]'
         para = block_from_string input, :backend => :docbook
         assert_equal '<inlineequation><mml:math xmlns:mml="http://www.w3.org/1998/Math/MathML"><mml:mi>a</mml:mi><mml:mo>&#x003C;</mml:mo><mml:mi>b</mml:mi></mml:math></inlineequation>', para.content
